parachain/primitives/beacon/src/updates.rs

```diff
@@ -86,14 +80,12 @@ pub struct FinalizedHeaderUpdate<const COMMITTEE_SIZE: usize, const COMMITTEE_BI
 	derive(Deserialize),
 	serde(deny_unknown_fields, bound(serialize = ""), bound(deserialize = ""))
 )]
-#[scale_info(skip_type_params(SyncCommitteeSize))]
-#[codec(mel_bound())]
 pub struct ExecutionHeaderUpdate<const COMMITTEE_SIZE: usize, const COMMITTEE_BITS_SIZE: usize> {
-	pub beacon_header: BeaconHeader,
-	pub execution_header: ExecutionPayloadHeader,
-	pub execution_branch: Vec<H256>,
+	pub attested_header: BeaconHeader,
```

Comment:
AFAIK we do execution sync from last sync to latest finalized so `beacon_header` in it should all be finalized,  so `attested_header` here does not make sense.

Reply:
It is true that the header is already consider finalized. However we are also verifying it using `verify_attested_header`,  as the sync committee has signed it. In that sense, I think it is good to be consistent with the rest of the code, and name it `attested_header`.